src/shaderlib/fp64/math-fp64.glsl

```diff
@@ -120,9 +122,9 @@ vec2 twoSum(float a, float b) {
 
 #if defined(NVIDIA_EQUATION_WORKAROUND) || defined(INTEL_EQUATION_WORKAROUND)
 vec2 twoSub(float a, float b) {
-  float s = (a - b) * ONE;
-  float v = (s - a);
-  float err = (a - (s - v) * ONE) * ONE - (b + v);
+  float s = (a - b);
+  float v = (s * ONE - a) * ONE;
+  float err = (a - (s - v) * ONE) * ONE * ONE * ONE - (b + v);
```

Comment:
Same here. There should perhaps be a small README on the GPU workaround package, with sections on Intel, Nvidia etc. We could reference that here.